Codeinwp/optimole-wp
# feat: add docs link to each control [ref: #621]

### All Submissions:

* x ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
 

### Changes proposed in this Pull Request:
 This adds docs link to each control as specified in the issue, and makes sure the visual hierarchy suggested by the design team is followed.

Closes #621.

### How to test the changes in this Pull Request:

1. Make sure each control that design team specified has the correct link the doc.
2. Make sure each control has the right title and description.
3. You can find the details here: https://www.notion.so/Optimole-descriptions-8467e37579ed45738905112323ed0f77

### Other information:
